packages/schematics/angular/no_typescript_runtime_dep_spec.js

```diff
@@ -21,9 +21,9 @@ describe('@schematics/angular javascript code', () => {
           const file = path.join(subdir, f);
           if (fs.statSync(file).isDirectory()) {
             check(file);
-          } else if (file.endsWith('.ts')) {
+          } else if (file.endsWith('.js')) {
             const content = fs.readFileSync(file, { encoding: 'utf-8' });
-            if (content.indexOf(`from 'typescript'`) >= 0) {
+            if (content.includes(`require("typescript")`) || content.includes(`require('typescript')`)) {
```

Comment:
Have you checked these imports in the generated files? Might be in UMD, although I think it shouldn't

Reply:
Yeah it’s a UMD.

Reply:
Does this test actually validates that there's no typescript import on UMDs?

Reply:
It’s validates that typescript is not imported/required.